src/esn.calendar.libs/app/services/event-service.js

```diff
@@ -497,7 +497,10 @@ function calEventService(
     if (!angular.isArray(event.attendees)) {
       return $q.when(null);
     }
-    if (!event.changeParticipation(status, emails)) {
+
+    if (event.organizer && _.isEqual(_.uniq(emails), [event.organizer.email])) {
```

Comment:
We can avoid to use lodash here for the easy check:
```suggestion
    if (event.organizer && emails.length === 1 && emails[0] === event.organizer.email) {
```